Tiltfile

```diff
@@ -0,0 +1,10 @@
+allow_k8s_contexts('kind-kind')
+analytics_settings(enable=False)
+
+load('ext://helm_remote', 'helm_remote')
+helm_remote('nats', repo_url='https://nats-io.github.io/k8s/helm/charts/', repo_name='nats', values='tilt-nats-values.yaml')
+
+custom_build('local-organic-guac', "export GUAC_IMAGE=\"$EXPECTED_IMAGE\" && make container", deps='pkg/', tag="latest")
```

Comment:
The build could be made faster if we have `goreleaser` only build images for the current architecture instead of all archs. Any ideas if this is possible and what flags we would set?

Reply:
yea - i do like this suggestion as i also have long build times. @sunnyyip what do you think of maybe having another make command or an ENV variable for building all archs?

Reply:
The container build runs on top of "release" which builds **bins** for all os/arch defined in .goreleaser.yaml and copy them into container images. 
goreleaser supports splitting builds on a per GOOS/GOARCH basis but it's a pro feature that won't be available for all users. 
Beyond that we have to edit the .goreleaser.yaml to override building only the current GOOS/GOARCH for each bin. i.e. use the builds.targets in https://goreleaser.com/customization/builds/. This can be messy as we have to manipulate the list of build configs in the yaml and will likely need additional tools installed.

As far as binary goes, we are already building for the host os/arch only using the "[--single-target](https://goreleaser.com/cmd/goreleaser_build/)" flag. 
One workaround I can think of is to use docker build and package the goreleaser output bins into the local container image. This won't be fast for the very first build since pulling the builder images and run build inside it takes time. Subsequent builds will be much faster tho.

Reply:
Thanks @sunnyyip - sounds like there's no easy way now. We can optimize this later :)